### PR TITLE
Image Rendering: Remove PhantomJS support

### LINKED ISSUE #13802: Remove PhantomJS
Tasks:
- [ ] Deprecate PhantomJS #18914
- [ ] Migration path for users upgrading when PhantomJS is removed? Deprecate first and give users time to switch to image renderer plugin and removing PhantomJS in the major/minor release after?
- [ ] Handle case when Grafana doesn't have any image renderer setup
- [ ] Blog post
- [ ] Update documentation

### PR DESCRIPTION
**Pull request by marefr**
**What this PR does / why we need it**:
Removes PhantomJS

- [x] Remove use of PhantomJS - step 1 #23375 
- [x] Verify custom docker images
- [x] Improve handling of image renderer not available/installed #23593
- [x] Breaking change and upgrading notes
- [x] Enterprise updates https://github.com/grafana/grafana-enterprise/pull/516
- [x] Merge and use new  version of build pipeline https://github.com/grafana/build-pipeline/pull/48

**Which issue(s) this PR fixes**:
Closes #13802

**Special notes for your reviewer**:
Improve UX for share panel/direct link rendered image when there are no renderer available/installed will be handled in a follow up PR.

### CODE REVIEW

**Comment by marefr** [comment]
For reviewers,  #23375 and #23593 have already been merged into this PR. This PR adds some custom docker build fixes and breaking change and upgrading notes. Idea is to handle required UX changes in a follow up PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 7.0.0 (unreleased)
+
+## Breaking changes
+
+- **Removal of PhantomJS**: PhantomJS was deprecated in [Grafana v6.4](https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/#phantomjs-deprecation) and starting from Grafana v7.0.0 PhantomJS support have been removed. This means that Grafana no longer ships with a built-in image renderer and we adwise you to install the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
```
Comment (oddlittlebird) [inline CHANGELOG.md:5]:
```suggestion
- **Removed PhantomJS**: PhantomJS was deprecated in [Grafana v6.4](https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/#phantomjs-deprecation) and starting from Grafana v7.0.0, all PhantomJS support has been removed. This means that Grafana no longer ships with a built-in image renderer, and we advise you to install the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
```

File: docs/sources/installation/upgrading.md [hunk 2]
```diff
@@ -238,3 +238,9 @@ Due to this change in Chrome, the `[security]` setting `cookie_samesite` configu
 
 This version of Chrome also rejects insecure `SameSite=None` cookies. See https://www.chromestatus.com/feature/5633521622188032 for more information. Make sure that you
 change the `[security]` setting `cookie_secure` to `true` and use HTTPS when `cookie_samesite` is configured to `none`, otherwise authentication in Grafana won't work properly.
+
+## Upgrading to v7.0
+
+### PhantomJS removed
+
+PhantomJS was deprecated in [Grafana v6.4](https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/#phantomjs-deprecation) and starting from Grafana v7.0.0 PhantomJS support have been removed. This means that Grafana no longer ships with a built-in image renderer and we adwise you to install the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
```
Comment (oddlittlebird) [inline docs/sources/installation/upgrading.md:246]:
```suggestion
PhantomJS was deprecated in [Grafana v6.4](https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/#phantomjs-deprecation) and starting from Grafana v7.0.0, all PhantomJS support has been removed. This means that Grafana no longer ships with a built-in image renderer, and we adwise you to install the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Please edit the [Image rendering](https://grafana.com/docs/grafana/latest/administration/image_rendering/) topic as well.

**Review by oddlittlebird — COMMENTED** [review]
LGTM!

**Review by oddlittlebird — APPROVED** [review]

**Review by aknuds1 — COMMENTED** [review]
LGTM!

**Pull request merged**